Dockerfile

```
@@ -10,10 +10,18 @@ RUN python -m venv /venv
 RUN /venv/bin/python -m pip install -U setuptools
 RUN /venv/bin/pip install -q .
 
-FROM reqs
+FROM reqs AS app
 COPY willa willa
 COPY README.rst README.rst
 COPY CHANGELOG.rst CHANGELOG.rst
 RUN /venv/bin/pip install .
 
 ENTRYPOINT ["/venv/bin/python"]
+
+FROM app AS development
+COPY --from=reqs /venv /venv
+COPY --from=reqs /app /app
+COPY tests tests
+RUN /venv/bin/pip install -q .[dev,lint]
```

`dev` includes `lint` (and `test`).

```suggestion
RUN /venv/bin/pip install -q .[dev]
```